server/routes/teamEventAPI.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const db = require("../db/models");
+const auth = require("../middleware/auth");
+
+const router = express.Router();
+
+// 1 - Create a new team event type
+router.post("/api/team-event", auth, (req, res) => {
+  if (req.body.duration) {
+    //
+    db.EventType.create({ ...req.body })
+      .then((response) => res.send(response))
+      .catch((error) => {
+        console.log(error);
+        res.status(500).send(error);
+      });
+  } else if (!req.body.duration) {
+    res.status(400).send("Duration is required");
+  }
+});
+
+// 2 - Add or remove members from an event type
+// an updated team member object is passed to DB to update added and removed members
+router.put("/api/team/:id", (req, res) => {
```

Comment:
In this case would it make more sense to search by team name instead?

Reply:
Probably not since we don't know if team name is unique